api/v1alpha1/types.go

```diff
@@ -67,31 +68,47 @@ type ImageBasedUpgradeSpec struct {
 	SeedImageRef SeedImageRef `json:"seedImageRef,omitempty"`
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Additional Images"
 	AdditionalImages ConfigMapRef `json:"additionalImages,omitempty"`
+	// OADPContent defines the list of ConfigMap resources that contain the OADP Backup and Restore CRs.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OADP Content"
 	OADPContent []ConfigMapRef `json:"oadpContent,omitempty"`
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Manifests"
+	// ExtraManifests defines the ConfigMap resource for the user-specific extra manifests in the Prep stage.
```

Comment:
Perhaps a more precise description?
```suggestion
	// ExtraManifests defines the list of ConfigMap resources that contain the user-specific extra manifests to be applied during the upgrade post-pivot stage .
```

Reply:
sure, added the text enhancement to #399